postgis/mvt.c

```diff
@@ -721,6 +721,74 @@ LWGEOM *mvt_geom(LWGEOM *lwgeom, GBOX *gbox, uint32_t extent, uint32_t buffer,
 }
 
 /**
+ * Transform a point geometry into vector tile coordinate space.
+ */
+LWGEOM *mvt_geom_point(LWPOINT *lwgeom, GBOX *gbox, uint32_t extent, uint32_t buffer,
+	bool clip_geom)
+{
+	POSTGIS_DEBUG(2, "mvt_geom called");
+	LWGEOM *lwgeom_out = NULL;
+	double width = gbox->xmax - gbox->xmin;
+	double height = gbox->ymax - gbox->ymin;
+	double resx = width / extent;
+	double resy = height / extent;
+	double fx = extent / width;
+	double fy = -(extent / height);
+	double buffer_map_xunits = resx * buffer;
+	double buffer_map_yunits = resy * buffer;
+	const GBOX *ggbox = lwgeom_get_bbox(lwgeom);
+
+	if (width == 0 || height == 0)
```

Comment:
Not knowing what GBOX can contain, Is it safer to check for <= 0 or force an absolute value when calculating width and height above?